.github/workflows/mlagility_validation.yml

```diff
@@ -0,0 +1,41 @@
+name: Validate created ONNX model from mlagility
+
+on:
+  push:
+    branches: [ main, new-models]
+  pull_request:
+    branches: [ main, new-models]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8']
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout repo
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: mla
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies and mlagility
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install onnx onnxruntime requests py-cpuinfo
```

Comment:
Line 36 installs both onnx and onnxruntime. Do we need a separate pip install here?

Reply:
I guess you are talking about this requirements: https://github.com/groq/mlagility/blob/main/models/requirements.txt, but actually onnx and onnxruntime are not there?

Reply:
Installation of MLAgility [here](https://github.com/groq/mlagility/blob/main/setup.py#L39) installs onnx and onnxruntime